docs/quick-start.md

```diff
@@ -13,9 +13,9 @@ Prior to installation, we recommend using [`Homebrew`](https://brew.sh/) to mana
 
 The Jetpack Monorepo requires the following to be installed on your machine:
 
-- Node.js (used for build process and our CLI): `nvm install`
-	- You can see the current latest supported version of Node.js in the monorepo root's [`package.json`](https://github.com/Automattic/jetpack/blob/master/package.json) under `engines`
-	- You can switch versions of node using nvm: `nvm ls` to list installed versions of node, `nvm use [version number]` to switch.
+- Bash: `brew install bash`
```

Comment:
Should we mention that, while people on Macs have bash, they have a too-old version so they'll need to do this anyway?